shared/search/user-pane/index.js

```diff
@@ -49,7 +49,7 @@ export default connector.connect(
       return {
         mode: 'external',
         nonUserInfoProps: {
-          avatar: userForInfoPane.serviceAvatar,
+          avatar: userForInfoPane.extraInfo.serviceAvatar || '',
```

Comment:
this pr was only supposed to be for the search results so this was just an unintended leftover